sktime/utils/_testing/series.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python3 -u
 # -*- coding: utf-8 -*-
 
-__author__ = ["mloning"]
+__author__ = ["mloning", "fkiraly"]
```

Comment:
Another place where I am not sure it makes sense to add @fkiraly!  I don't have an issue with you guys adding him, but again, would need to get his permission before we merge this if that is what you guys want to do! :)

Reply:
Reverted it. Thank you.